src/index.html

```diff
@@ -1,13 +1,425 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Miami Condo Kings</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="icon" href="images/favicon.png">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__section header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a
+              href="#menu"
+              class="icon icon--menu">
+            </a>
+
+            <a href="#home" class="logo">
+              <img
+                src="images/logo.png"
+                alt="Miami Condo Kings logo"
+                class="logo__image">
+            </a>
+
+            <a
+              href="tel: +1 234 555-55-55"
+              class="icon icon--phone header__call">
+            </a>
+          </div>
+
+          <h1 class="header__title">
+            Miami
+            <br>
+            Condo Kings
+          </h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top top-actions">
+            <a
+              href="#"
+              class="icon icon--cross">
+            </a>
+
+            <a href="#home" class="logo">
+              <img
+                src="images/logo.png"
+                alt="Miami Condo Kings logo"
+                class="logo__image">
+            </a>
+          </div>
+
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link">Home</a>
+            </li>
+            <li class="menu__item">
+              <a href="#about-us" class="menu__link">About us</a>
+            </li>
+            <li class="menu__item">
+              <a href="#gallery" class="menu__link">Gallery</a>
+            </li>
+            <li class="menu__item">
+              <a href="#testimonials" class="menu__link">Testimonials</a>
+            </li>
+            <li class="menu__item">
+              <a href="#contact-us" class="menu__link">Contact us</a>
+            </li>
+          </ul>
+
+          <a
+            href="tel:+1 234 555-55-55"
+            class="menu__call menu__call--phone">
+            +1 234 555-55-55
+          </a>
```

Comment:
Should fix that for all file

```suggestion
          <a
            href="tel:+1 234 555-55-55"
            class="menu__call menu__call--phone"
          >
            +1 234 555-55-55
          </a>
```